app/controllers/repos_controller.rb

```diff
@@ -85,8 +83,12 @@ def parse_params_for_repo_info
       if params[:url]
         params[:url].match(/^https:\/\/github\.com\/([^\/]*)\/([^\/]*)\/?$/)
         params[:repo] ||= {}
-        params[:repo][:user_name] = $1
-        params[:repo][:name] = $2
+        params[:repo][:user_name] = $1.to_s
+        params[:repo][:name] = $2.to_s
```

Comment:
Didn't realize this match logic was in here. We don't need to do it now, but we should eventually switch to a match block and named matches

```ruby
params[:repo] ||= {}
params[:url].match(/^https:\/\/github\.com\/(?<user_name>[^\/]*)\/(?<name>[^\/]*)\/?$/) do |m|
  params[:repo][:user_name] = m[:user_name]
  params[:repo][:name]      = m[:name]
end
```